test/saithrift/tests/sail2.py

```diff
@@ -379,3 +379,93 @@ def runTest(self):
             self.client.sai_thrift_set_port_attribute(port2, attr)
             self.client.sai_thrift_set_port_attribute(port3, attr)
             self.client.sai_thrift_set_port_attribute(port4, attr)
+			
+class L2SonicVlanTest(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
+        print "[access vlan=10]"
+        print "1. Sending L2 broadcast packet from port 1 -> all vlan members."
+        print "2. Sending L2 unicast packet from port 1 -> port 2."
+        switch_init(self.client)
+        default_vlan = 1
+        vlan_id = 10
+        default_vlan_members_list = []
+        port1 = port_list[0]
```

Comment:
make 24 ports? we also need to verify the other 8 ports do not receive the packet